.editorconfig

```diff
@@ -22,3 +22,6 @@ max_line_length = unset
 
 [{src/ClientInterface.php,src/ClientContextInterface.php}]
 max_line_length = unset
+
+[tests/Predis/Command/Redis/Search/FTEXPLAIN_Test.php]
+indent_size = unset
```

Comment:
Let's not add exceptions for test files.

Reply:
@tillkruss Well, in this case CI breaks, to make sure that expected response exactly matches actual response there should be no indents in it

Reply:
`eclint` is so inflexible 😓

Reply:
We may have 1 additional exception per year!!!

Reply:
@vladvildanov Can you try adding a 2nd `.editorconfig` to the `/tests` folder that contains the `FTEXPLAIN_Test.php` exception only?

Reply:
@tillkruss Works!